### DISCUSSION TITLE
Dark Mode: 404 Page

### DISCUSSION THREAD

**Discussion by oscard0m** [community/community#3482] [Discussions] [answered]
Seems like the 404 page should have more contrast when using Dark Mode:

Example: https://github.com/oscard0m/a

![image](https://user-images.githubusercontent.com/2574275/110117333-fe5e9d00-7db8-11eb-861f-b98be20797d6.png)

Not sure if this should be posted here or there is a better place to issue this

**Comment by PranamBhat** [comment, marked as answer]
@oscard0m Good catch! I request you to post the same over :

[https://github.community/](https://github.community/)

When I opened the link you provided, it is working fine:


![image](https://user-images.githubusercontent.com/32120033/110126841-dd16a480-7dea-11eb-84b9-ca68f678cc29.png)

**Reply by oscard0m** [reply]
Thanks @PranamBhat !
I tested in Incognito Mode and it works so... the issue is [this Chrome extension](https://github.com/sindresorhus/refined-github)! When I disable it, it works as expected

I just opened [an issue](https://github.com/sindresorhus/refined-github/issues/4062) to them and we can close this discussion.

Thanks for your time and sorry for the confusion here!

**Reply by PranamBhat** [reply]
**Perfect**! Kindly mark this as "**Answered**" and close this discussion.

**Reply by oscard0m** [reply]
> and close this discussion.

Not sure on how to `close` this discussion... @PranamBhat

**Reply by PranamBhat** [reply]
@oscard0m You can mark as "**Answered**". That's all :)

**Reply by oscard0m** [reply]
To add more info. It is a feature from the extension, not a bug 💪🏾

https://github.com/sindresorhus/refined-github/pull/4048
